src/main/java/at/gammastrahlung/monopoly_server/network/websocket/MonopolyMessageHandler.java

```diff
@@ -176,21 +201,15 @@ public static ServerMessage joinGame(int gameId, WebSocketPlayer player) {
     public static ServerMessage startGame(WebSocketPlayer player) {
         Game game = player.getCurrentGame();
 
-        if (game.startGame(player))
-            return generateUpdateMessage(ServerMessage.MessageType.SUCCESS, game.getState());
-        else
-            return generateUpdateMessage(ServerMessage.MessageType.ERROR, game.getState());
+        if (game.startGame(player)) return generateUpdateMessage(ServerMessage.MessageType.SUCCESS, game.getState());
+        else return generateUpdateMessage(ServerMessage.MessageType.ERROR, game.getState());
     }
 
     private static ServerMessage initiateRound(WebSocketPlayer player) {
         Game game = player.getCurrentGame();
         Player currentPlayer = game.getCurrentPlayer();
 
-        ServerMessage message = ServerMessage.builder()
-                .messagePath("initiate_round")
-                .type(ServerMessage.MessageType.INFO)
-                .jsonData(gson.toJson(currentPlayer))
-                .build();
+        ServerMessage message = ServerMessage.builder().messagePath("initiate_round").type(ServerMessage.MessageType.INFO).jsonData(gson.toJson(currentPlayer)).build();
```

Comment:
please revert this, it's hard to read and not an improvement, makes retention worse for no reason